CHANGES.rst

```diff
@@ -1834,6 +1834,10 @@ Other Changes and Additions
 - Fixing ``astropy.__citation__`` to provide the full bibtex entry of the 2018
   paper. [#8110]
 
+- Adding ``warning_type`` keyword argumentum to the deprecated decorators to
```

Comment:
```suggestion
- Adding ``warning_type`` keyword argument to the "deprecated" decorators to
```

Reply:
The quotes to signify that the decorators themselves are not deprecated, but rather they help deprecate things.